src/components/shared/Entity/hooks/useEntityCreateNavigate.ts

```diff
@@ -45,10 +51,15 @@ export default function useEntityCreateNavigate() {
                 ? appendSearchParams(searchParamConfig)
                 : null;
 
-            const newPath: string = advanceToForm
+            let newPath: string = advanceToForm
                 ? ENTITY_SETTINGS[entity].routes.createNew
                 : ENTITY_SETTINGS[entity].routes.connectorSelect;
 
+            if (entity === 'capture' && expressWorkflow && advanceToForm) {
+                // TODO (powered-by-estuary): Use an error page as a fallback.
+                newPath = ENTITY_SETTINGS[entity].routes.createNewExpress;
+            }
```

Comment:
So I think needing this does answer a thing we discussed at the start... the `entity_settings` will need _some_ variability in it. Don't change anything now.